src/data/stream/basic.lean

```diff
@@ -1,23 +1,523 @@
 /-
-Copyright (c) 2020 Gabriel Ebner, Simon Hudon. All rights reserved.
+Copyright (c) 2015 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Gabriel Ebner, Simon Hudon
+Authors: Leonardo de Moura
 -/
+import data.stream.defs
 import tactic.ext
-import data.stream.init
 
 /-!
-# Additional instances and attributes for streams
--/
+# Streams a.k.a. infinite lists a.k.a. infinite sequences
+
+This file used to be in the core library. It was moved to `mathlib` and renamed to `init` to avoid
+name clashes.  -/
```

Comment:
```suggestion
Most of this file used to be in the core library. -/
```
Can we rename this file to `data/stream.lean`? Or does that ruin the diff?

Reply:
I'm going to add more to `data.stream` folder, so I'd prefer to leave this as `basic`.

Reply:
The diff will work out better if you rename the file as I suggest in this commit, and rename it back to `basic` later when you add the other files.

Reply:
I moved back to `data.stream.init`.